### PR TITLE
[search] A fix to group translations.

### PR DESCRIPTION
**Pull request by mpimenov**

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -646,16 +646,16 @@ el:Αρτοποιείο|κατάστημα
 sk:3Pekáreň|obchod
 sw:Kituo cha kuwoka mikate
 
-shop
+shop|@shop
```
Comment (ygorshenin) [inline data/categories.txt]: Could you please explain this self-reference?

Comment (ygorshenin) [inline data/categories.txt]: Or, at least, move prefix-length specifiers to the '@shop' and get rid of duplicates here.

File: data/categories.txt [hunk 2]
```diff
@@ -2360,148 +2360,148 @@ da:Monument|seværdigheder
...
 el:Αρχεία|θέαμα|τουρισμός|αξιοθέατα
 sk:Zrúcanina|ruiny|pamätihodnosť
 sw:Maeneo ya kale|jengo la kale|chakavu
 
-historic-castle
+historic-castle|@tourism
 en:Castle|attraction|tourism|sights|U+1F3EF|U+1F3F0|U+1F451|U+1F478
 ru:Замок|достопримечательность|туризм
-ar:قلعةأماكن الجذب|سياحة|مناظر
+ar:قلعةأماكن الجذب|مناظر
```
Comment (ygorshenin) [inline data/categories.txt]: Does it worth to move 'туризм' to '@tourism' ?

File: data/categories.txt [hunk 3]
```diff
@@ -2599,100 +2599,100 @@ el:Ενοικίαση αυτοκινήτων|αυτοκίνητο|ενοικιά
 sk:3Autopožičovňa|požičovňa áut
 sw:Kodi ya gari|gari|kodi
 
-amenity-cinema
+amenity-cinema|@entertainment
```
Comment (ygorshenin) [inline data/categories.txt]: Again, as you reference '@entertainment' here, please, get rid of duplicates. Here, and in other cases.

File: data/categories.txt (hunk 1 above)
Comment (mpimenov) [inline data/categories.txt]: shop is an osm key
@shop contains translations for keys from the shop category

"shop" belongs to the "@shop" category

Comment (mpimenov) [inline data/categories.txt]: Duplicates removed.

File: data/categories.txt (hunk 2 above)
Comment (mpimenov) [inline data/categories.txt]: @tourism is translated as "Sights" not as "Tourism" in our UI.

File: data/categories.txt (hunk 3 above)
Comment (mpimenov) [inline data/categories.txt]: done

**Comment by mpimenov** [comment]
PTAL

File: data/categories.txt [hunk 4]
```diff
@@ -646,36 +646,36 @@ el:Αρτοποιείο|κατάστημα
...
-sv:2Butik
-th:2ร้านค้า
-tr:2Mağaza
+shop|@shop
+en:U+1F3EA|U+1F3EC
+ru:
+ar:متجر
+cs:
+da:forretning
+nl:
```
Comment (ygorshenin) [inline data/categories.txt]: You don't need these empty fiels.

Comment (mpimenov) [inline data/categories.txt]: Removed.

File: data/categories.txt [hunk 5]
```diff
@@ -434,170 +434,170 @@ zh-Hant:1匯率|理財|外幣|兌換|貨幣
 el:Συνάλλαγμα|ανταλλακτήριο|χρήματα
 sk:3Zmenáreň|peniaze
 
-amenity-bar
-en:2Bar|2pub|beer|drink|food|U+1F37A|U+1F37B|U+1F376
+amenity-bar|@food
+en:2Bar|2pub|beer|drink|U+1F37A|U+1F37B|U+1F376
```
Comment (ygorshenin) [inline data/categories.txt]: I do not understand why en:Food disappeared, but ru:еда is left.

Comment (mpimenov) [inline data/categories.txt]: tolower doesn't work with non-ascii

**Comment by therearesomewhocallmetim** [comment]
LGTM

**Comment by mpimenov** [comment]
Merge, please.

**Pull request merged**